docs/samples/calling/app.js

```diff
@@ -575,7 +594,7 @@ async function getMediaStreams() {
 function createMeeting(e) {
   e.preventDefault();
 
-  callingClient = window.callingClient = Calling.createClient(webex);
+  callingClient = window.callingClient = calling.callingClient;
```

Comment:
same here

Reply:
We are not creating client when we do calling.callingClient, we are only accessing it multiple times and storing in window interface. But yeah it makes sense that we can do this once at global level. Made the change